src/services/legal-services.ts

```diff
@@ -1,9 +1,10 @@
 import { axios } from '@/utils/'
 import { AddressesIF, AlterationFilingIF, BusinessInformationIF, ChgRegistrationFilingIF, ConversionFilingIF,
-  CorrectionFilingIF, NameTranslationIF, OrgPersonIF, ResolutionsIF, ShareStructureIF, SpecialResolutionFilingIF }
+  CorrectionFilingIF, NameTranslationIF, OrgPersonIF, ResolutionsIF, SpecialResolutionFilingIF }
   from '@/interfaces/'
 import { RoleTypes } from '@/enums'
 
+import { ShareStructureIF } from '@bcrs-shared-components/interfaces/'
```

Comment:
Yes, you've now seen that we don't re-export the shared interfaces in this UI. This was to avoid some cyclic imports at one time. I'm not sure if that still exists but I'm totally OK with each component importing local or shared imports (as long as we don't do it both ways simultaneously).